### ISSUE TITLE
Uncaught Error: EEXIST: file already exists, mkdir 'C:\Users\VOXIWEB\Documents\FTP\indicatifs'

### ISSUE THREAD

**Issue by natinusala** [icetee/remote-ftp#354]
[Enter steps to reproduce below:]
1. Create a file named "indicatifs" in the project directory
2. Create a new folder named "indicatifs" in the remote FTP
3. Open this folder from the remote tree view

**Atom Version**: 1.7.3
**System**: Unknown Windows Version
**Thrown From**: [Remote-FTP](https://github.com/mgrenier/remote-ftp) package, v0.8.1
### Stack Trace

Uncaught Error: EEXIST: file already exists, mkdir 'C:\Users\VOXIWEB\Documents\FTP\indicatifs'

```
At C:\Users\VOXIWEB\AppData\Local\atom\app-1.7.3\resources\app.asar\node_modules\fs-plus\node_modules\mkdirp\index.js:76

Error: EEXIST: file already exists, mkdir 'C:\Users\VOXIWEB\Documents\FTP\indicatifs'
    at Error (native)
    at fs.mkdirSync (fs.js:842:18)
    at Object.fs.mkdirSync (ATOM_SHELL_ASAR.js:592:16)
    at Function.sync (C:\Users\VOXIWEB\AppData\Local\atom\app-1.7.3\resources\app.asar\node_modules\fs-plus\node_modules\mkdirp\index.js:55:12)
    at Object.fsPlus.makeTreeSync (C:\Users\VOXIWEB\AppData\Local\atom\app-1.7.3\resources\app.asar\node_modules\fs-plus\lib\fs-plus.js:343:23)
    at C:\Users\VOXIWEB\.atom\packages\Remote-FTP\lib\connectors\ftp.js:447:8
    at Object.cb (C:\Users\VOXIWEB\.atom\packages\Remote-FTP\node_modules\ftp\lib\connection.js:1017:13)
    at Parser.<anonymous> (C:\Users\VOXIWEB\.atom\packages\Remote-FTP\node_modules\ftp\lib\connection.js:117:20)
    at emitTwo (events.js:87:13)
    at Parser.emit (events.js:172:7)
    at Parser._write (C:\Users\VOXIWEB\.atom\packages\Remote-FTP\node_modules\ftp\lib\parser.js:59:10)
    at doWrite (_stream_writable.js:292:12)
    at writeOrBuffer (_stream_writable.js:278:5)
    at Parser.Writable.write (_stream_writable.js:207:11)
    at Socket.ondata (C:\Users\VOXIWEB\.atom\packages\Remote-FTP\node_modules\ftp\lib\connection.js:273:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)
```
### Commands

```
     -0:32.8.0 remote-ftp:delete-selected (span.name.icon.icon-file-directory)
     -0:25.1.0 remote-ftp:refresh-selected (div.header.list-item)
     -0:23.7.0 remote-ftp:add-folder (div.header.list-item)
  3x -0:20.7.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:14.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:11.4.0 remote-ftp:add-file (span.name.icon.icon-file-directory)
     -0:08.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "packagesWithKeymapsDisabled": [
      "tag"
    ],
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
autocomplete-php, v0.3.7
drunken-php, v0.2.1
linter, v1.11.4
linter-php, v1.2.0
minimap, v4.23.2
minimap-cursorline, v0.1.0
minimap-find-and-replace, v4.5.1
php-getters-setters, v0.5.5
Remote-FTP, v0.8.1
selector-to-tag, v0.4.1

# Dev
No dev packages
```

**Comment by Keeland** [comment]
Also getting this error.

**Comment by nmanley** [comment]
I encountered the same issue. I found that a file without an extension that was named the same and the target directory or its descendants was in my local directory which is a minor-problem within windows itself. I removed the local file, disconnected and reconnected the ftp connection, and it worked like a charm.

Other symptoms includes: 
File will not download, because its unable to create a local directory in windows.
Un-able to create remote file without this Uncaught error.

Hope this helps.
